scripts/vcpkg/get_mumble_dependencies.ps1

```diff
@@ -6,29 +6,29 @@ $mumble_deps = "qt5-base",
                "qt5-tools",
                "grpc",
                "boost-accumulators",
-               "opus",
                "poco",
                "libvorbis",
                "libogg",
                "libflac",
                "libsndfile",
                "libmariadb",
-               "zlib",
+               "mdnsresponder",

```

Comment:
Why was that not needed before? Isn't that a dependency only? :eyes:

Reply:
I don't recall removing zlib, so it must have been accidental.

Reply:
mdnsresponder was installed without `--clean-after-build` before when we were doing each package individually. It was moved to the array.

Reply:
I just saw that we used to install `icu` as well but this doesn't seem to have been moved up here. Is this just not needed?

Reply:
`qt5-base` marks it during install.